src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveReviewsServiceTest.kt

```diff
@@ -286,6 +339,50 @@ class IncentiveReviewsServiceTest {
     )
   }
 
+  @Test
+  fun `throw exception if cannot calculate next review date for one bookingId`(): Unit = runBlocking {
```

Comment:
Unless I'm missing something this test is now for missing _**reviews**_ (`PrisonerIepLevel` records) rather than missing/uncalculatable next review dates.

I think the test description should reflect this to avoid any confusion.

Reply:
quite right! i forgot to rename